ksp_plugin_test/benchmark.cpp

```diff
@@ -76,26 +134,15 @@ void BM_PluginDeserializationBenchmark(benchmark::State& state) {
   int bytes_processed = 0;
   for (auto _ : state) {
     PushDeserializer* deserializer = nullptr;
-    Plugin const* plugin = nullptr;
-    int l = 1;
-    for (auto const& line : gipfeli_plugin) {
-      principia__DeserializePluginHexadecimal(line.c_str(),
-                                              line.size(),
-                                              &deserializer,
-                                              &plugin,
-                                              compressor);
-      bytes_processed += line.size() >> 1;
-    }
-    principia__DeserializePluginHexadecimal("",
-                                            0,
-                                            &deserializer,
-                                            &plugin,
-                                            compressor);
+    Plugin const* plugin = DeserializePluginFromLines(gipfeli_plugin,
+                                                      compressor,
+                                                      bytes_processed);
     principia__DeletePlugin(&plugin);
```

Comment:
Should this be after a `PauseTiming` so we don't benchmark destruction?

Reply:
Irrelevant after the other change.